parsers/ENTSOE.py

```diff
@@ -134,10 +134,14 @@
                    ENTSOE_DOMAIN_MAPPINGS['DK-DK1']],
     'DE->DK-DK2': [ENTSOE_DOMAIN_MAPPINGS['DE-AT-LU'],
                    ENTSOE_DOMAIN_MAPPINGS['DK-DK2']],
-    'DE->SE-SE4': [ENTSOE_DOMAIN_MAPPINGS['SE-SE4'],
-                   ENTSOE_DOMAIN_MAPPINGS['DE-AT-LU']],
+    'DE->SE-SE4': [ENTSOE_DOMAIN_MAPPINGS['DE-AT-LU'],
```

Comment:
@corradio I guess that mean s the DE->SE-SE4 had the wrong sign, right?

Reply:
That could be, but not sure. Let's open an issue to double check.